rero_ils/config.py

```diff
@@ -449,7 +449,7 @@ def _(x):
         list_route='/collections/',
         item_route='/collections/<pid(coll, record_class="rero_ils.modules.collections.api:Collection"):pid_value>',
         default_media_type='application/json',
-        max_result_window=10000,
+        max_result_window=20000,
```

Comment:
It would be nice to use a variable for this.

Reply:
ok